draft-irtf-cfrg-sigma-protocols.md

```diff
@@ -91,7 +91,7 @@ This document describes interactive sigma protocols, a class of secure, general-
 
 # Introduction
 
-Any sigma protocol must define three objects: a *commitment* (computed by the prover), a *challenge* (computed by the verifier), and a *response* (computed by the prover).
+Any sigma protocol must define three objects: a *commitment* (computed by the prover), a *challenge* (computed by the verifier), and a *response* (computed by the prover). One of the advantages of sigma protocols is their composability which enables the construction of more complex protocols (e.g., OR, AND, t-out-of-n composition). However, such compositions must be handled carefully to preserve security properties as discussed in {{sec-cons}}.
```

Comment:
Instead of just mentioning the different types of composition, I would say a bit about what they mean (in a sentence or two). That way readers with no background on sigma protocols can make sense of this.

To keep things simple, I would focus on just one type of composition. We've spent a lot of time discussing OR, so that's the one I'd go with.

Finally, I would add a reference for this. Perhaps the [Boneh-Shoup book](http://crypto.stanford.edu/~dabo/cryptobook/)? There are examples of this in the rest of the file :)